### ISSUE TITLE
.detach()

### ISSUE THREAD

**Issue by saiinc** [GyverLibs/SoftServo#1]
После вызова myservo.detach(), myservo.attach(pin) больше не работает.

**Comment by GyverLibs** [comment]
проверил, работает

**Comment by saiinc** [comment]
Похоже проблема в попытке использования мною myservo.read(), серва начинает вести себя непредсказуемым образом. Не посмотрел сначала, что этого метода нету в readme. Правда в readme есть getCurrentDeg() и intread() в английской версии, но их нет в самой библиотеке.

**Comment by GyverLibs** [comment]
английская версия является автоматическим переводом, там должно быть предупреждение

**Comment by saiinc** [comment]
Как тогда получить текущую позицию в градусах? В README.md для этого указан int getCurrentDeg(), но при компиляции ошибка: 'class SoftServo' has no member named 'getCurrentDeg'.

**Comment by GyverLibs** [comment]
Опа, извиняюсь, в ридми затесалась документация из другой библиотеки. Read тут есть
// вернуть текущий угол
int read()

**Comment by GyverLibs** [comment]
внёс некоторые правки, скорее всего проблема с детач должна пропасть. Доку обновил, read() был и есть в реализации
